docs/docs/visual-testing-with-storybook.md

```diff
@@ -143,6 +137,15 @@ npm run storybook
 
 Storybook CLI adds this command to your `package.json` for you so you shouldn't have to anything other than run the command. If Storybook builds successfully you should be able to navigate to `http://localhost:6006` and see the default stories supplied by the Storybook CLI.
 
+However, if you use `StaticQuery` or `useStaticQuery` in your project Storybook needs to be run with the `NODE_ENV` set to `production` (as Storybook sets this by default to `development`). Otherwise `babel-plugin-remove-graphql-queries` won't be run. Moreover Storybook needs to know about [static files](https://storybook.js.org/docs/configurations/serving-static-files/#2-via-a-directory) generated by Gatsby's `StaticQuery`. Your scripts should look like:
+
+```json:title=package.json
+{
+  "storybook": "NODE_ENV=production start-storybook -s static",
```

Comment:
Maybe add these to a top-level scripts? e.g.

```json
{
  "scripts": {
    "storybook": "NODE_ENV=production start-storybook -s static",
    "build-storybook": "NODE_ENV=production build-storybook -s static"
  }
}
```

Reply:
@DSchau Done 👍